answer/ponpain/HTML/02/anchor.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>anchor</title>
+</head>
+<body>
+    <h1>同一タブ</h1>
+    <a href="https://isc-makeit.github.io/Make-IT-homepage/">Make It</a>
+    <h1>新規タブ</h1>
+    <a href="https://isc-makeit.github.io/Make-IT-homepage/" target="_blank">Make It</a>
+    <h1>JS</h1>
+    <script type="text/javascript">
+        function openwin() {
+        window.open('https://isc-makeit.github.io/Make-IT-homepage/',"_blank",);
```

Comment:
indentしましょうか.
openに渡すものはカンマのあとは空白。末尾のカンマは不要かと 👀 

```suggestion
            window.open('https://isc-makeit.github.io/Make-IT-homepage/', "_blank");
````